src/test/java/uk/gov/ch/developer/docs/controller/developer/UserCallbackControllerTest.java

```diff
@@ -17,6 +9,11 @@
 import uk.gov.ch.oauth.Oauth2;
 import uk.gov.ch.oauth.identity.IdentityProvider;
 
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import static org.mockito.Mockito.*;
```

Comment:
Coding standards say that we can't have `.*` imports.

Reply:
I would have expected the code formatting to expand this.

Reply:
IntelliJ is doing something weird here, I have expanded them but it keeps grouping the imports